infra/feast-operator/api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -452,6 +452,17 @@ func (in *RegistryFilePersistence) DeepCopyInto(out *RegistryFilePersistence) {
 		*out = new(PvcConfig)
 		(*in).DeepCopyInto(*out)
 	}
+	if in.S3AddtlKwargs != nil {
+		in, out := &in.S3AddtlKwargs, &out.S3AddtlKwargs
+		*out = new(map[string]string)
```

Comment:
this has been reassigned in the deep copy using make. Do we need to initialize with empty map?

Reply:
this file is generated by `make generate`, I assume it is corrrect 😉